index.src.html

```diff
@@ -3998,19 +4027,29 @@
             [=combined buffer layout/allocationSize=].
     9. Return |combinedLayout|.
 
+
+: <dfn for=VideoFrame>Copy VideoFrame metadata</dfn> (with |metadata|)
+:: 1. Let |metadataCopySerialized| be [$StructuredSerialize$](|metadata|).
```

Comment:
I still think we want `forStorage` here so that the resulting serialized data can be held long-term in an opaque format (I'm imagining that Chrome's implementation would want to hold the data in the underlying resource, so that it can flow through the media engine). Do you happen to know which WG we can ask for advice on that?

Reply:
Is there a use case for actually storing VideoFrame in things like IDB?

I thought of a potential use case where you could assign some serialisable but not forStorage objects (like a CropTarget) to a VideoFrame metadata.

Reply:
FWIW, VideoFrame current algorithm does not allow serialisation for IDB (it throws when forStorage is true).

Reply:
Removed user metadata for now